README.md

```diff
@@ -27,6 +27,19 @@ It is also possible to compile _libygg_ into an RPM suitable for installation on
 Fedora-based distributions. See the README in [dist/srpm](./dist/srpm) for
 details.
 
+### Python stubs
+
+_libygg_ can be used in Python as a `gi.repository` module (see the `examples/` directory).
+
+You can install the type stubs into your virtual environment:
```

Comment:
How do these stubs get installed? Does the `python/` directory need to include a `meson.build` that installs the Python files? Does `Ygg.pyi` get written out somewhere to the filesystem?

Reply:
After running `pip install`, the file `Ygg.pyi` is saved to `{{ VENV_DIR }}/lib/python3.12/site-packages/gi-stubs/repository/Ygg.pyi`, where the type checkers can use it.

No idea about meson, I have no experience there.

Reply:
Ok, so when `meson install` is run then, this file needs to get written into `$python_site_packages/gi-stubs/repository`. Maybe the `python` object's `get_install_dir` method could get us to the right location? https://mesonbuild.com/Python-module.html#get_install_dir. `pip` isn't used at all when compiling and installing with meson, so we have to make sure that files that are needed by Python at runtime are installed when `meson install` gets run.

Reply:
In theory, they aren't needed at runtime at all. But it is good idea to install the stubs with the package itself; in theory we could just leave it as-is, installable on-demand with pip, though.

Reply:
I see. Do we know if other Fedora packages include these stub files and install them?

Reply:
There's a handful of stub files that get packaged by various Fedora packages. They seem to be installed into directories next to where their corresponding ".py" file would go, so to allow the system Python to support these type hints, we should install them too.